src/Tasks.kt

```diff
@@ -0,0 +1,10 @@
+package test.agileFreaks
+
+fun IntArray.sumOfEven():Int {
```

Comment:
Why refactor? What was wrong with the first implementation?

Reply:
The reason I've refactored it it's the complexity and the unnecessary reiteration that was done after filtering the elements of the array, that one was executing two for loops, one to filter all the elements and another one to sum the remaining elements.